core/templates/services/extension-tag-assembler.service.ts

```diff
@@ -84,13 +87,15 @@ export class ExtensionTagAssemblerService {
       element: JQuery,
       customizationArgs: InteractionCustomizationArgs
   ): JQuery {
+    const caBackendDict = <Record<string, {value: Object}>> (
+      this._convertCustomizationArgsToBackendDict(customizationArgs));
     for (const caName in customizationArgs) {
-      const caBackendDict = (
-        this._convertCustomizationArgsToBackendDict(customizationArgs));
-      let caBackendDictValue = caBackendDict[caName].value;
-      element.attr(
-        this.camelCaseToHyphens.transform(caName) + '-with-value',
-        this.htmlEscaperService.objToEscapedJson(caBackendDictValue));
+      if ('value' in caBackendDict[caName]) {
```

Comment:
Is this `if` statement redundant? Can you try running the checks once after removing these ?

Reply:
Oh correct, done.

Reply:
Not done